src/extension.ts

```diff
@@ -94,6 +95,10 @@ function bundleRspecLastExecuted() {
   }
 }
 
+function getRSpecCommand(): string {
+  return vscode.workspace.getConfiguration().get(SETTINGS_RSPEC_COMMAND_KEY);
```

Comment:
```suggestion
let configCommand = vscode.workspace
    .getConfiguration()
    .get(SETTINGS_RSPEC_COMMAND_KEY);

  return configCommand || "bundle exec rspec --color";
```

Reply:
So we don't need this based on my comment above, thanks =)